var/spack/repos/builtin/packages/protobuf/package.py

```diff
@@ -16,6 +16,7 @@ class Protobuf(CMakePackage):
     url = "https://github.com/protocolbuffers/protobuf/archive/v3.18.0.tar.gz"
     maintainers("hyoklee")
 
+    version("3.22.2", sha256="2118051b4fb3814d59d258533a4e35452934b1ddb41230261c9543384cbb4dfc")
```

Comment:
Protobuf decided that 2 versioning schemes wasn't confusing enough, so now they simultaneously release 22.2, 4.22.2, and 3.22.2. As far as I can tell, these are all the same, but I could be wrong.